src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/static/json_schema.json

```
@@ -83,13 +83,13 @@
       "type": "boolean",
       "title": "Mark vertical lines",
       "default": false,
-      "description": "States whether to reproduce vertical lines in the document."
+      "description": "States whether to reproduce vertical lines in the document. Note: This parameter is not applicable if `mode=native_text`."
     },
     "mark_horizontal_lines": {
       "type": "boolean",
       "title": "Mark horizontal lines",
       "default": false,
-      "description": "States whether to reproduce horizontal lines in the document."
+      "description": "States whether to reproduce horizontal lines in the document. Note: This parameter is not applicable if `mode=native_text` and will not work if `mark_vertical_lines` is set to `false`."
```

```suggestion
      "description": "States whether to reproduce horizontal lines in the document. Note: This parameter is not applicable if mode chosen is `native_text` and will not work if `mark_vertical_lines` is set to `false`."
```

[SUGGESTION] Do you think we can make it this way? Feel free to resolve if otherwise.